autogen/header.rs

```diff
@@ -109,18 +110,19 @@ fn gen_value_enum_operand_kind(grammar: &structs::OperandKind) -> TokenStream {
         } else {
             // Special case for Dim. Its enumerants can start with a digit.
             // So prefix with the kind name here.
-            let name = if grammar.kind == "Dim" {
+            let name_str = if grammar.kind == "Dim" {
                 let mut name = "Dim".to_string();
                 name.push_str(&e.symbol);
                 name
             } else {
                 e.symbol.to_string()
             };
-            let name = as_ident(&name);
+            let name = as_ident(&name_str);
             let number = e.value;
             seen_discriminator.insert(e.value, name.clone());
             enumerants.push(quote! { #name = #number });
             from_prim_list.push(quote! { #number => #kind::#name });
+            from_str_impl.push(quote! { #name_str => Ok(#kind::#name), });
```

Comment:
Could this be

```suggestion
            from_str_impl.push(quote! { #name_str => Ok(Self::#name), });
```

To save quite a bunch of characters on every line?